code/modules/reagents/newchem/toxins.dm

```diff
@@ -555,6 +555,40 @@ datum/reagent/tabun/on_mob_life(var/mob/living/M as mob)
 	..()
 	return
 
+/datum/reagent/chloralhydrate
+	name = "Chloral Hydrate"
+	id = "chloralhydrate"
+	description = "A powerful sedative, Lethal in larger doses."
+	reagent_state = LIQUID
+	color = "#000067" // rgb: 0, 0, 103
+
+	on_mob_life(var/mob/living/M as mob)
+		if(!M) M = holder.my_atom
+		if(!data) data = 1
+		data++
+		switch(data)
+			if(1)
+				M.confused += 2
+				M.drowsyness += 2
+			if(2 to 50)
+				M.sleeping += 1
+			if(51 to INFINITY)
+				M.sleeping += 1
+				if(iscarbon(M))
+					var/mob/living/carbon/C = M
+					C.toxloss += (data - 50)
```

Comment:
You should probably be using the `apply_damage` proc instead of increasing toxloss directly.